x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -118,6 +117,10 @@ export class HeadlessChromiumDriverFactory {
           },
         } as LaunchOptions);
 
+        // log the child process PID
+        const childProcess: ChildProcess = browser.process();
```

Comment:
This is 2/2 of the actual changes. Everything else is due to increase strictness of Typescript